### PR TITLE
Add tests for TC-SC-5.1/2

### PR DESCRIPTION
**Pull request by cecille**
Adding test automation for TC-SC-5.1, TC-SC-5.2, also add key material to match.

per https://github.com/CHIP-Specifications/chip-test-plans/pull/2344

### CODE REVIEW

**Comment by cletnick** [comment]
LGTM

**Review by jmeg-sfy — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_SC_5_2.yaml [hunk 1]
```diff
@@ -17,46 +17,141 @@ name: 19.1.2. [TC-SC-5.2] Receiving a group message - TH to DUT
...
+      groupId: 0x0103
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0103
+      response:
+          values:
+              - name: Status
+                value: 0
+              - name: GroupId
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_SC_5_2.yaml:108]: I would expect the name here (and similar places) to be "GroupID"...

File: src/app/tests/suites/certification/Test_TC_SC_5_1.yaml [hunk 2]
```diff
@@ -13,257 +13,226 @@
...
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0103
+      response:
+          values:
+              - name: Status
+                value: 0
+              - name: GroupId
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_SC_5_1.yaml:126]: Would expect the name here and in similar places to be GroupID?

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_SC_5_1.yaml (hunk 2 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_SC_5_1.yaml]: Yeah, GroupID for view groups, GroupId in key management :) . Either way, the zapt does some fancy case changes.

File: src/app/tests/suites/certification/Test_TC_SC_5_2.yaml (hunk 1 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_SC_5_2.yaml]: changed in next. Doesn't affect the generated files since zap does a case change.

**Pull request merged**